ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -285,6 +293,7 @@ querySupportedVersion = \case
     GetLedgerTip                               -> (>= v1)
     GetEpochNo                                 -> (>= v1)
     GetNonMyopicMemberRewards {}               -> (>= v1)
+    GetRewardProvenance                        -> (>= v2)
```

Comment:
@mrBliss is it okay to add this new query to version 2 like I've done? Is there anywhere else that I need to document it?

Reply:
That makes sense, thank you for the explanation!